src/contrib/mlir/core/pass/mlir_subgraph_extraction.cpp

```diff
@@ -531,7 +529,7 @@ bool MLIRSubgraphExtractionPass::is_supported_mlir_op(std::shared_ptr<Node> node
     if (auto softmax = as_type_ptr<ngraph::op::Softmax>(node))
     {
         // Softmax is only supported through callback
-        if (!getenv_bool("NGRAPH_MLIR_CALLBACK"))
+        if (std::getenv("NGRAPH_MLIR_CALLBACK") == nullptr)
```

Comment:
Why don't we use the `getenv_bool` utility?

Reply:
will create another PR to update getenv not related to convBias for CPU backend.